src/safeds/data/image/containers/_image.py

```diff
@@ -506,22 +544,7 @@ def rotate_left(self) -> Image:
         result : Image
             The image rotated 90 degrees counter-clockwise.
         """
-        image_copy = copy.deepcopy(self)
-        image_copy._image = image_copy._image.rotate(90, expand=True)
-        return image_copy
-
-    def find_edges(self) -> Image:
-        """
-        Return a grayscale version of the image with the edges highlighted.
-
-        The original image is not modified.
+        return Image(func2.rotate(self._image_tensor, 90, expand=True), device=self.device)
 
-        Returns
-        -------
-        result : Image
-            The image with edges found.
-        """
-        image_copy = copy.deepcopy(self)
-        image_copy = image_copy.convert_to_grayscale()
-        image_copy._image = image_copy._image.filter(ImageFilter.FIND_EDGES)
-        return image_copy
+    # def find_edges(self) -> Image:
+    #     pass
```

Comment:
Remove + add an issue. Otherwise, we'll forget about it.

Reply:
see #523